Cargo.toml

```diff
@@ -3,6 +3,7 @@
 members = [
   "tower",
   "tower-balance",
+  "tower-blocking-service",
```

Comment:
I think this could just live in tower itself

Reply:
I agree.

Reply:
Moved it into tower.